.travis.yml

```diff
@@ -0,0 +1,17 @@
+language: java
```

Comment:
add the commands to run the tests

Reply:
@kaweesi  Have addressed and updated the pull request